test/cli/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ cmake_minimum_required (VERSION 3.11)
 add_cli_test(iGenVar_cli_test.cpp)
 add_dependencies (iGenVar_cli_test "iGenVar")
 target_use_datasources (iGenVar_cli_test FILES simulated.minimap2.hg19.coordsorted_cutoff.sam)
+target_use_datasources (iGenVar_cli_test FILES mini_example_reference.fasta)
```

Comment:
Oh, I think this file needs to be updated, because I think I only updated the explanation fasta the last time I expanded my example.... 🙈

Reply:
I will update it in my next PR, where I worked on this anyways